### PR TITLE
Grafting disclaimer

### PR DESCRIPTION
**Pull request by MichaelMacaulay**

### CODE REVIEW

File: website/pages/en/cookbook/grafting.mdx [hunk 1]
```diff
@@ -24,6 +24,22 @@ For more information, you can check:
 
 In this tutorial, we will be covering a basic usecase. We will replace an existing contract with an identical contract (with a new address, but the same code). Then, graft the existing subgraph onto the "base" subgraph that tracks the new contract.
 
+## Important Note on Grafting When Upgrading to the Network
+
+> Caution: If you are migrating your subgraph from Subgraph Studio or the hosted service to the decentralized network, it is strongly recommended to avoid using grafting during the upgrade process.
+
+### Why is this Important?
```
Comment (benface) [inline website/pages/en/cookbook/grafting.mdx:31]: Are "is" and "this" not capitalized in title case?

File: website/pages/en/developing/creating-a-subgraph.mdx [hunk 2]
```diff
@@ -934,6 +934,8 @@ If the subgraph encounters an error, that query will return both the data and a
 
 ### Grafting onto Existing Subgraphs
 
+> **Note:** it is not recommended to use grafting when initially upgrading to The Graph Network. For more information, learn more [here](/cookbook/grafting/#important-note-on-grafting-when-upgrading-to-the-network).
```
Comment (benface) [inline website/pages/en/developing/creating-a-subgraph.mdx:937]: Maybe just me, but "For more information, learn more [here]" sounds a bit redundant to me. How about simply "More information [here]" or "Learn more [here]"?

Comment (benface) [reply website/pages/en/developing/creating-a-subgraph.mdx:937]: Also, super nitpicky, but in the callouts above, you didn't use bold for "Disclaimer" and "Caution" but here you used it for "Note", and also above you capitalized the word after the `:` and here you didn't.

File: website/pages/en/cookbook/grafting.mdx (hunk 1 above)
Comment (MichaelMacaulay) [reply website/pages/en/cookbook/grafting.mdx:31]: No - that's a typo - thank you!

File: website/pages/en/developing/creating-a-subgraph.mdx (hunk 2 above)
Comment (MichaelMacaulay) [reply website/pages/en/developing/creating-a-subgraph.mdx:937]: Agreed - fixing that!

**Review by benface — APPROVED** [review]

**Comment by GundamDweeb** [comment]
This question might be irrelevant in time as hosted service is slated to be sunset, but is grafting from decentralized network to hosted/studio possible? Might want to mention something about that too.

**Pull request merged**